src/main/target/STM32F3DISCOVERY/target.h

```diff
@@ -36,8 +36,11 @@
 #define LED1                    PE10 // Orange LEDs - PE10/PE14
 #define LED1_INVERTED
 
-#define BEEPER                  PE9  // Red LEDs - PE9/PE13
-#define BEEPER_INVERTED
+//#define BEEPER                  PE9  // Red LEDs - PE9/PE13
+//#define BEEPER_INVERTED
+
+#define BEEPER_PWM              PE11
+#define BEEPER_PWM_FREQ         2200 //fréquence en Hz
```

Comment:
The comment is easy to understand, but english would be better
Also using BEEPER_PWM_HZ will carry unit information for free ...